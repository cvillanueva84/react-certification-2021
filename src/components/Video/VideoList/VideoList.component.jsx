import React from 'react';
import { videoData } from '../../../youtube-videos-mock';
import { Container } from './VideoList.styled';
import VideoItem from '../VideoItem/VideoItem.component';

export default function VideoList() {
  const { items } = videoData;

  const avatar = items[0].snippet.thumbnails.default.url;

  return (
    <Container>
      {items.slice(1).map((item, id) => {
        const {
          snippet: {
            thumbnails: {
              medium: { url },
            },
          },
        } = item;

        const {
          snippet: { title },
        } = item;

        const {
          snippet: { description },
        } = item;

        return (
          <VideoItem
            key={id}
            url={url}
            title={title}
            description={description}
            avatar={avatar}
          />
        );
      })}
    </Container>
  );
}
