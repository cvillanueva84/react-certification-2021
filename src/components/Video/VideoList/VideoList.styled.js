import styled from 'styled-components';

export const Container = styled.section`
  margin-top: 100px;
  position: relative;
  z-index: 0;
  width: 800px;

  @media screen and (min-width: 768px) {
    display: grid;
    grid-template-columns: 1fr 1fr;
    grid-column-gap: 1.5rem;
  }

  @media screen and (min-width: 992px) {
    grid-template-columns: repeat(3, 1fr);
  }
`;
