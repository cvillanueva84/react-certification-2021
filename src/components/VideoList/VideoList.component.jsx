/* eslint-disable no-nested-ternary */
import React, { useContext, useEffect } from 'react';
import styled from 'styled-components';
import { useHistory } from 'react-router-dom';

import VideoCard from '../VideoCard';
import VideoDetailsView from '../VideoDetailsView';

import { useFetchVideos } from '../../utils/hooks/useFetchVideos';
import { StateContext } from '../../context/State/state';

const CardListContainer = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  padding: 0 70px 70px 70px;
`;

const VideoList = () => {
  const stateContext = useContext(StateContext);
  const { location } = useHistory();
  const {
    video: { search, openedDetails },
    handleVideosList,
    handleSelectVideo,
    toggleOpenedDetails,
  } = stateContext;
  const { data } = useFetchVideos(search);

  const handleOpenDetails = (item) => {
    handleSelectVideo(item);
    toggleOpenedDetails(openedDetails);
  };

  const getVideoPath = (item) => {
    if (location.pathname.includes('favorites')) {
      return `/favorites/${item.id.videoId}`;
    }
    return `/${item.id.videoId}`;
  };

  useEffect(() => {
    handleVideosList(data);
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [data]);

  useEffect(() => {
    handleVideosList(data);
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [search]);

  return openedDetails ? (
    <VideoDetailsView getVideoPath={getVideoPath} />
  ) : (
    <>
      {location.pathname.includes('favorites') ? (
        <h1 style={{ textAlign: 'center' }}>Favorites!</h1>
      ) : (
        <h1>Welcome to the Challenge!</h1>
      )}
      <CardListContainer>
        {data ? (
          data.error ? (
            <div>Error Fetching data...</div>
          ) : (
            data.items.map((item, idx) => (
              <VideoCard
                // eslint-disable-next-line react/no-array-index-key
                key={idx}
                description={item.snippet.description}
                title={item.snippet.title}
                image={item.snippet.thumbnails.medium.url}
                handleOpenDetails={handleOpenDetails}
                item={item}
                getVideoPath={getVideoPath}
              />
            ))
          )
        ) : (
          <div>Loading...</div>
        )}
      </CardListContainer>
    </>
  );
};

export default VideoList;
