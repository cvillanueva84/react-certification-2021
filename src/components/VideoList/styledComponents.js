import styled from 'styled-components';

export const VideoCardDiv = styled.div`
  display: flex;
  flex-direction: column;
  width: 30rem;
  height: 35rem;
  border-radius: 0.4rem;
  margin: 2rem;
  border: 1px #dad4d4 solid;
  transition: all 0.3s ease;
  &:hover {
      box-shadow: 10px 10px 24px 0px rgba(0, 0, 0, 0.75);
    }
    
    @media only screen and (max-width: 670px) {
        width: 80%;
    }
`;
VideoCardDiv.displayName = 'VideoCardDiv';

export const VideoImageDiv = styled.div`
  height: 140px;
`;
VideoImageDiv.displayName = 'VideoImageDiv';

export const TitleSpan = styled.span`
  font-size: 1.6rem;
  font-weight: bold;
  margin: 2rem;
`;
TitleSpan.displayName = 'TitleSpan';

export const DescriptionSpan = styled.span`
  font-size: 1.4rem;
  margin-bottom: 2rem;
  text-align: left;
  margin: 2rem;
  `;
DescriptionSpan.displayName = 'DescriptionSpan';

