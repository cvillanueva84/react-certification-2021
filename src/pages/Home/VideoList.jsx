import React from 'react';
import styled from 'styled-components';
import VideoItem from './VideoItem';

const VideoListContainer = styled.div`
    width: 100%;
    height: 100vh;
    margin: 5px auto;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    
    & a {
        text-decoration: none;
    }
`;

const VideoList = ({videos}) => {

    if (videos.length < 0 ) return null;

    return (
        <VideoListContainer>
            {videos.items.map( (item) => (
                <VideoItem item={item} key={item.etag} />
            ))}
        </VideoListContainer>
    );

};

 export default VideoList;

 