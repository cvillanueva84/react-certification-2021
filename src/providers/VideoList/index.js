export {  VideoListProvider } from './VideoList.provider';
