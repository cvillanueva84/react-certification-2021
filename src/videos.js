const videos = [
    {
        id: 1,
        title: 'Video 1',
        descripcion: 'Información sobre video 1',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },
    {
        id: 2,
        title: 'Video 2',
        descripcion: 'Información sobre video 2',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },
    {
        id: 3,
        title: 'Video 3',
        descripcion: 'Información sobre video 3',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },
    {
        id: 4,
        title: 'Video 4',
        descripcion: 'Información sobre video 4',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 5,
        title: 'Video 5',
        descripcion: 'Información sobre video 5',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 6,
        title: 'Video 6',
        descripcion: 'Información sobre video 6',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 7,
        title: 'Video 7',
        descripcion: 'Información sobre video 7',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 8,
        title: 'Video 8',
        descripcion: 'Información sobre video 8',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 9,
        title: 'Video 9',
        descripcion: 'Información sobre video 9',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 10,
        title: 'Video 10',
        descripcion: 'Información sobre video 10',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 11,
        title: 'Video 11',
        descripcion: 'Información sobre video 11',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 12,
        title: 'Video 12',
        descripcion: 'Información sobre video 12',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 13,
        title: 'Video 13',
        descripcion: 'Información sobre video 13',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 14,
        title: 'Video 14',
        descripcion: 'Información sobre video 14',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'    
    },{
        id: 15,
        title: 'Video 15',
        descripcion: 'Información sobre video 15',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 16,
        title: 'Video 16',
        descripcion: 'Información sobre video 16',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 17,
        title: 'Video 17',
        descripcion: 'Información sobre video 17',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 18,
        title: 'Video 18',
        descripcion: 'Información sobre video 18',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 19,
        title: 'Video 19',
        descripcion: 'Información sobre video 19',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 20,
        title: 'Video 21',
        descripcion: 'Información sobre video 21',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 22,
        title: 'Video 22',
        descripcion: 'Información sobre video 23',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 24,
        title: 'Video 24',
        descripcion: 'Información sobre video 24',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    },{
        id: 25,
        title: 'Video 25',
        descripcion: 'Información sobre video 25',
        img: 'https://images.pexels.com/photos/1476880/pexels-photo-1476880.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
    }

]

export default videos;