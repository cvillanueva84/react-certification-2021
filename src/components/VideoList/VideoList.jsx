import React, { useEffect, useState } from 'react';
import axios from 'axios';
import { DivTitle, Container, Fakegrid } from './VideoList.styled';
import Card from '../Card/Card';

const baseURL =
  'https://gist.githubusercontent.com/jparciga/1d4dd34fb06ba74237f8966e2e777ff5/raw/f3af25f1505deb67e2cc9ee625a633f24d8983ff/youtube-videos-mock.json';

const VideoList = () => {
  const [videos, setVideos] = useState([]);

  useEffect(() => {
    async function getVideos() {
      const {
        data: { items },
      } = await axios.get(baseURL);
      const aux = [...items];
      aux.shift();
      setVideos(aux);
      // console.log(data.items.map((el) => el.snippet.description));
    }
    getVideos();
  }, []);

  return (
    <>
      <DivTitle>
        <h2>Welcome to the Challenge!</h2>
      </DivTitle>
      <Container>
        <Fakegrid>
          {videos.map((video) => (
            <Card
              title={video.snippet.title}
              description={video.snippet.description}
              link={video.snippet.thumbnails.high.url}
              key={video.snippet.publishedAt}
            />
          ))}
        </Fakegrid>
      </Container>
    </>
  );
};

export default VideoList;
