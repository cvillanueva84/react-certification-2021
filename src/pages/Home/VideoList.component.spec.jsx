import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import VideoList from './VideoList.component';
import videos from '../../mock/youtube-videos-mock.json';

describe('VideoList component', () => {
  it('should match snapshot', () => {
    const snap = renderer.create(<VideoList videos={videos} />);

    expect(snap.toJSON()).toMatchSnapshot();
  });

  it('should render a list of videos', () => {
    render(<VideoList videos={videos} />);

    const listItems = screen.getAllByRole('link');

    expect(listItems.length).toBeGreaterThan(0);
  });
});
