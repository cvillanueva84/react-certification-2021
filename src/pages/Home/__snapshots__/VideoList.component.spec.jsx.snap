// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoList component should match snapshot 1`] = `
<div
  className="sc-gKAaRy cTjvxd"
>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi cAICiJ"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Wizeline transforms how teams build technology. Its customers accelerate the delivery of innovative products with proven solutions, which combine Wizeline's ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi ckEFay"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Video Tour | Welcome to Wizeline Guadalajara
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi nfApA"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline Guadalajara | Bringing Silicon Valley to Mexico
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Wizeline continues to offer a Silicon Valley culture in burgeoning innovation hubs like Mexico and Vietnam. In 2018, our Guadalajara team moved into a ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi cRtYNo"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline hace sentir a empleados como en casa
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          En el 2014, Bismarck fundó Wizeline, compañía tecnológica que trabaja con los corporativos ofreciendo una plataforma para que desarrollen software de forma ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi kqYTVB"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          We Are Wizeline
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Engineering a better tomorrow. Wizeline is a global software development company that helps its clients solve their biggest challenges with design and ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi logoGn"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline Thrives in Mexico City
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Our vibrant Mexico City office is home to agile software engineers, talented UX designers, and brilliant data scientists. Learn about the rich history of Mexico City.
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi ieZPEN"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          El plan de Wizeline, una empresa de inteligencia artificial, para ayudar a crecer la comunidad de ciencia de datos en CDMX y todo el país, a través de cursos ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi gvRWsi"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          A Day in the Life of an Engineering Manager at Wizeline
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Fernando Espinoza shares his experience working as an engineering manager at Wizeline and mentoring other engineers. Learn about Fernando's passions ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi euHMDl"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline Technical Writing Academy | Featuring Eduardo Ocejo
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi bPTmYU"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Silicon Valley en México
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Empresas de Silicon Valley buscan establecerse en México por el gran talento que hay en nuestro país. Es una investigación de Roberto Domínguez.
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi jFWbMX"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Why Wizeline? featuring Juan Pablo Villa in Mexico City
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Juan Pablo, known as Gianpa at Wizeline, is a software engineer in our Mexico City office. Gianpa focuses on Android apps, is an integral part of our culture, ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi bMPtgA"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline&#39;s 2020 Year in Review
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          There's no doubt that 2020 has been an unprecedented year. However, amidst all the chaos, we achieved remarkable growth in various areas of our business.
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi gLjOhp"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Why Wizeline? featuring Hugo Lopez in Mexico City
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi kYubRY"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Why Wizeline? (We&#39;re Hiring in Mexico!)
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          A quick look at why people join Wizeline, what motivates us as a team and what it's like to work in our Guadalajara office. Learn more and apply here: ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi dCImnR"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Why Wizeline? featuring Oswaldo Herrera in Mexico City
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Oswaldo is a software engineering in Wizeline's Mexico City office. He joined Wizeline because of the camaraderie and deep sense of commitment of our teams.
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi hRjUsd"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Caso de Éxito AWS: Wizeline [Spanish]
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Central de socios de APN - https://amzn.to/2S7tIXM Fundada en 2014, Wizeline es una compañía joven e innovadora que nació en la nube para ofrecer soporte ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi dpjAuU"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline&#39;s 2019 Year in Review
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi cDPPzV"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          [1 of 2] Wizeline CEO shares career lessons from Google
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Founder & CEO Bismarck Lepe on growth opportunities at Wizeline and his career-path experience as an early Google employee. Join our team!
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi VOMiN"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline Data Engineering featuring Tania Reyes
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Tania discovered her interest in Big Data while working at Wizeline and took Wizeline Academy courses to skill up and join the data team. Now, she works on ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi dTdVKz"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Wizeline Querétaro | Mexico&#39;s New Knowledge Economy (We&#39;re hiring!)
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          A small but mighty (and growing) team, the Queretaro crew has taken ownership of growing the office and brand, speaking at university events, hosting tech ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi DkTAr"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Miriam Godinez | Women in Leadership at Wizeline
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Science and technology always caught Miriam's attention. One of her ultimate goals as a Senior Engineer Manager and Lead from the Mobile Team at Wizeline ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi LxXrJ"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Welcome Back to Wizeline Vietnam | Extended Version
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Thanks to swift government action, the COVID-19 situation in Vietnam has reached a point where businesses are able to return to work and reopen offices.
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi kKVKjZ"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          How does Wizeline work?
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Wizeline builds teams with a mix of technical and non-technical talent to deliver better products, faster. Learn more about our consulting services: ...
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi jfwXHb"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          12 Wishes from Wizeline | Happy Holidays 2019
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          
        </p>
      </div>
    </div>
  </div>
  <div
    role="link"
  >
    <div
      className="sc-gtsrHT NaRnq"
    >
      <div
        className="sc-dlnjwi OHUvt"
        role="img"
      />
      <div
        className="sc-hKFxyN eaXTjP"
      >
        <h3
          className="sc-eCApnc eSfhDM"
        >
          Why Wizeline?
        </h3>
        <p
          className="sc-jSFjdj edDWTs"
          role="note"
        >
          Hear from our employees directly about what excites them about their roles here at Wizeline. Wizeline wants to hire the best and the brightest to accelerate their ...
        </p>
      </div>
    </div>
  </div>
</div>
`;
